packages/client/fixtures/blog/prisma/schema.prisma

```diff
@@ -5,7 +5,8 @@ datasource db {
 
 generator client {
   provider        = "prisma-client-js"
-  previewFeatures = ["napi", "selectRelationCount"]
+  previewFeatures = ["selectRelationCount"]
+  engineType = "binary"
```

Comment:
is this necessary here?

Reply:
No this is just a fixture used for development

Reply:
So not a test? Might be useful to rename this then from `blog` to `blog-used-during-development` or something like that.